### ISSUE TITLE
Update aws-node CNI Version in eksctl utils

### ISSUE THREAD

**Issue by dschunack** [weaveworks/eksctl#4037]
**What feature/behavior/change do you want?**
Please, update eksctl to rollout aws-node v1.8.0 or higher. At the moment v1.7.10 will be installed/updated with eksctl utils update-aws-node. 

**Why do you want this feature?**
Problem is that v1.7.10 had a Bug and all of our EKS are affected by this. This bug is fixed in v1.8.0 or newer. 
Maybe it make also sense to add a --version parameter to use a specific version. Good to test new versions.

**Comment by steveizzle** [comment]
We are  facing the same Issue. AWS released already  v1.9.0. Will you provide newer Versions for the update of aws-node with eksctl utils?

```
eksctl utils update-aws-node --cluster eks-test --approve

2021-08-12 10:04:39 [ℹ]  eksctl version 0.60.0
2021-08-12 10:04:39 [ℹ]  using region eu-central-1
2021-08-12 10:04:42 [ℹ]  replaced "ClusterRoleBinding.rbac.authorization.k8s.io/aws-node"
2021-08-12 10:04:42 [ℹ]  replaced "ClusterRole.rbac.authorization.k8s.io/aws-node"
2021-08-12 10:04:43 [ℹ]  replaced "CustomResourceDefinition.apiextensions.k8s.io/eniconfigs.crd.k8s.amazonaws.com"
2021-08-12 10:04:43 [ℹ]  replaced "kube-system:DaemonSet.apps/aws-node"
2021-08-12 10:04:43 [ℹ]  skipped existing "kube-system:ServiceAccount/aws-node"
2021-08-12 10:04:43 [ℹ]  "aws-node" is now up-to-date

 k describe ds -n kube-system aws-node | grep "k8s-cni:"
    Image:      602401143452.dkr.ecr.eu-central-1.amazonaws.com/amazon-k8s-cni:v1.7.10

```

**Comment by aclevername** [comment]
Thanks for raising the issue. I've created a PR to bump eksctl to `1.9`https://github.com/weaveworks/eksctl/pull/4085
